movies/presentation/src/main/java/com/example/movies/presentation/ui/moviesList/MoviesListView.kt

```diff
@@ -0,0 +1,172 @@
+                                    .fillMaxSize(),
+                                contentAlignment = Alignment.Center
+                            ) {
+                                Column() {
+                                    CircularProgressIndicator()
+                                    Text(text = "Loading...")
+                                }
+                            }
+                        }
+                    }
+                }
+                listState.OnBottomReached(buffer = 1) {
+                    viewModel.getMoviesList()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MovieItem(movie: Movie?) {
+    Card(
+        modifier = Modifier
+            .padding(8.dp, 4.dp)
+            .fillMaxWidth()
+            .height(100.dp),
+        shape = RoundedCornerShape(8.dp),
+        elevation = 4.dp
+    )
+    {
+        Surface {
+            Row(
+                Modifier
+                    .padding(4.dp)
+                    .fillMaxSize()
+            ) {
+
+                Image(
+                    painter = rememberAsyncImagePainter(
+                        ImageRequest.Builder(LocalContext.current)
+                            .data(data = movie?.getCompleteUrlToDetails())
+                            .apply(block = fun ImageRequest.Builder.() {
+                                scale(Scale.FIT)
+                                placeholder(R.drawable.movie_placeholder)
+                            }).build()
+                    ),
+                    contentDescription = movie?.overview,
+                    modifier = Modifier
+                        .fillMaxHeight()
+                        .weight(0.2f)
+                )
+
+                Column(
+                    verticalArrangement = Arrangement.Center,
+                    modifier = Modifier
+                        .padding(4.dp)
+                        .fillMaxHeight()
+                        .weight(0.8f)
+                ) {
+                    Text(
+                        text = movie?.name ?: "",
+                        style = MaterialTheme.typography.subtitle1,
+                        fontWeight = FontWeight.Bold
+                    )
+                    Text(
+                        text = movie?.score.toString(),
+                        style = MaterialTheme.typography.caption,
+                        fontWeight = FontWeight.Bold,
+                        modifier = Modifier
+                            .background(
+                                Color.LightGray
+                            )
+                            .padding(4.dp)
+                    )
+                    Text(
+                        text = movie?.overview ?: "",
+                        style = MaterialTheme.typography.body1,
+                        maxLines = 2,
+                        overflow = TextOverflow.Ellipsis
+                    )
+
+                }
+            }
+        }
+    }
+}
+
```

Comment:
Please, add this composable to a new file

Reply:
Moved, done.